docs/content/latest/yugabyte-platform/troubleshoot/universe-issues.md

```diff
@@ -15,29 +15,242 @@ isTocNested: true
 showAsideToc: true
 ---
 
-## Metrics Page
+Yugabyte Platform allows you to monitor and troubleshoot issues that arise from universes.
 
-In the [Admin Console](../../install-yugabyte-platform/), select the [Universe](../../../architecture/concepts/universe/) page, then go to the **Metrics** tab. The page shows a number of interactive metrics graphs that capture the state of the Yugabyte Universe over time.
+## Use Metrics
 
-![Yugabyte Metrics Page](/images/troubleshooting/check-metrics.png)
+A universe's **Metrics** page displays graphs representing information on operations, latency, and other parameters accumulated over time for each type of node and server. Examining specific metrics allows you to diagnose and troubleshoot issues.
 
-Note: For a quick overview, check the query ops and latency graphs as well as the CPU, memory, disk, and network usage graphs. In case of dips or spikes the other graphs will offer additional information that can help diagnose the issue.
+You access metrics by navigating to **Universes > Universe-Name > Metrics**, as shown in the following illustration:
 
-## Nodes Status
+![Metrics Page](/images/yp/metrics.png)
 
-In the [Admin Console](../../install-yugabyte-platform/), click on the [Universe](../../../architecture/concepts/universe/) page, then go to the **Nodes** tab.
-The page will show the status of the Master and TServer on each YugabyteDB node.
+You should do the following on a regular basis:
 
-![Yugabyte Nodes Page](/images/troubleshooting/check-node-status.png)
+- Monitor remote bootstrap for node failures. You can access this metric via **Tablet Server > Remote Bootstraps** and **Tablet Server > RPC Queue Size**.
+- Monitor the NTP server to ensure that it is running. You can access this information via your computer's operating system.
+- Monitor clock skew so you can see when the clock is off by more than 500 milliseconds. You can access this metric via **Node > Clock Skew**.
+- Monitor your disk usage. The default is 80% used, but you should consider this metric in the context of your environment. For example, the usage percentage can be higher on larger disks; some file systems issue an alert at 75% usage due to performance degradation. You can access this metric via your computer's operating system.
+- Monitor YSQL and YCQL operations latency and tune it according to your application service level agreement. You can access this metrics via **YSQL Ops and Latency**, **YCQL Ops and Latency**, **YSQL Advanced**, and **YSQL Advanced**. 
```

Comment:
Last one should be **YCQL Advanced**

Reply:
Yes, thanks for catching it. Unfortunately, I can't cancel the merge -- will fix in a few minutes in a new PR

Reply:
fixed